extension/intents/music/music.toml

```diff
@@ -152,3 +152,6 @@ match = """
 
 [[music.showTitle.example]]
 phrase = "What is playing"
+
+[[music.showTitle.example]]
+phrase = "What's playing?"
```

Comment:
This could be `test = true` since it's very similar to the other example

Reply:
Confirming `test = true` has been added for this example.

Reply:
Yes, this is right